src/chat.js

```diff
@@ -0,0 +1,39 @@
+import io from 'socket.io-client';
+import 'jquery';
+
+export default class Chat {
+    constructor(nick) {
+        this.nick = nick;
+        this.socket = io({query: `nick=${nick}`});
+
+        if (__DEV__)
```

Comment:
I would put the block into curly brackets, or delete the new line after the condition. :D